ansible/roles/edu-sharing/templates/mirror-vocabs.sh.j2

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
 {% for mirror in mirror_vocabularies %}
-wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
-chown -R www-data:www-data {{ mirror.target_directory }}
+    wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
+    chown -R www-data:www-data {{ mirror.target_directory }}
```

Comment:
line-format is changed, but no other update. Please keep the format of unchanged lines

Reply:
is ready